packages/shared/src/components/cards/squad/UnfeaturedSquadGrid.tsx

```diff
@@ -10,39 +10,41 @@ import {
 } from '../../typography/Typography';
 import { largeNumberFormat } from '../../../lib';
 import { Separator } from '../common';
-import { SquadImage } from './common/SquadImage';
-import { SquadJoinButtonWrapper } from './common/SquadJoinButton';
 import { UnFeaturedSquadCardProps } from './common/types';
+import { Origin } from '../../../lib/log';
+import { SquadActionButton } from '../../squads/SquadActionButton';
+import { ButtonVariant } from '../../buttons/common';
+import { Image, ImageType } from '../../image/Image';
 
 export const UnfeaturedSquadGrid = ({
   source,
-  title,
-  subtitle,
-  icon,
-  action,
-  description,
+  className,
 }: UnFeaturedSquadCardProps): ReactElement => {
+  const router = useRouter();
+  const title = source.name;
+
   return (
-    <Card className={classNames('min-w-[19rem] overflow-hidden border-0 p-4')}>
+    <Card className={classNames('overflow-hidden border-0 p-4', className)}>
       <CardLink
         href={source.permalink}
         rel="noopener"
         title={source.description}
       />
       <div className="mb-3 flex items-center justify-between">
-        <SquadImage
-          image={source?.image}
-          icon={icon}
-          title={title}
-          size="size-16"
+        <Image
+          src={source.image}
+          alt={`${title} source`}
+          className="size-16 rounded-full"
+          type={ImageType.Squad}
         />
```

Comment:
Nothing urgent, but we should really introduce a better image wrapper.
Also think we support Size as a enum on these afaik.

(Maybe something for OS?)

Reply:
> Also think we support Size as a enum on these afaik.

Not on this image component

> Nothing urgent, but we should really introduce a better image wrapper.

Yeah could be an interesting task. I'll save and let Lee know about this